pages/funds/index.js

```diff
@@ -1,3 +1,48 @@
+import {useCallback, useContext, useEffect, useState} from "react";
+import CoinCard from "../../components/CoinCard";
+import {CMContext} from "../../context/context";
+import styled from "styled-components";
+
 export default function Funds() {
-  return <p>Page Funds is under construction</p>;
+  let {getCoins} = useContext(CMContext);
+  let [coinData, setCoinData] = useState(null);
+
+  useEffect(() => {
+    setData();
+    /* eslint-disable-next-line react-hooks/exhaustive-deps */
+  }, []);
+
+  const setData = useCallback(async () => {
+    try {
+      let apiResponse = await getCoins();
+      setCoinData(apiResponse);
+    } catch (error) {
+      console.log(error.message);
+    }
+  }, [getCoins]);
```

Comment:
If you put this part inside the useEffect, you won't need to disable eslint.

Reply:
useEffect does not allow you to add async in the callback function, as google said

Reply:
You can not directly change the callback function to a async function put you can put an async function inside the callback function of the useeffect.